ccdaservice/unservice.js

```diff
@@ -0,0 +1,18 @@
+var isWin = /^win/.test(process.platform);
+if( isWin ){
+	var Service = require('node-windows').Service;
```

Comment:
The winservice/unservice run as expected in windows however in Linux, the current setup is winservice starts in root and is not seen by OpenEMR running above root so it runs as a process. Prefer method would be to run in root and I keep going back and forth as to best way to handle. Thus the two methods. I haven't worried too much on how to stop it, several ways to do. Long time since I've looked at background service code but there, I assume, is where to enable feature. Remembering that if CCM and/or Portal is enabled, so must ccda service. I'll figure this out.